feat(alerting): custom alert support endpoint errors (#844)
TwiN/gatus: README.md

- `[ENDPOINT_NAME]` (resolved from `endpoints[].name`)
- `[ENDPOINT_GROUP]` (resolved from `endpoints[].group`)
- `[ENDPOINT_URL]` (resolved from `endpoints[].url`)

> - `[RESULT_ERRORS]` (resolved from the endpoint's health check output in `endpoint.Result.Errors[]`)

Reviewer: ```suggestion
- `[RESULT_ERRORS]` (resolved from the health evaluation of a given health check)
```
